crypto/ec/ec_backend.c

```diff
@@ -157,6 +157,11 @@ int ec_group_todata(const EC_GROUP *group, OSSL_PARAM_BLD *tmpl,
             goto err;
 
         if (fid == NID_X9_62_characteristic_two_field) {
+#ifdef OPENSSL_NO_EC2M
+            ECerr(EC_F_EC_ASN1_GROUP2FIELDID, EC_R_GF2M_NOT_SUPPORTED);
+            goto err;
+        }
+#else
```

Comment:
Please don't do that kind of wrapping one brace but not the other.  Simply leave that ending brace *outside* the wrapping.

Reply:
Ok fixed in another place also.